Library/Homebrew/manpages/brew.1.md

```diff
@@ -298,6 +298,10 @@ Note that these flags should only appear after a command.
 
     If no <formulae> are given, check all installed brews.
 
+  * `migrate` [<formulae>]:
```

Comment:
I haven't studied this PR closely, but it it looks like what this does is check if a rename is registered in `FORMULA_RENAMES`, then perform the necessary migrations?

This seems like how an internal tool might work, not something we're presenting to users.  When I saw there was a `brew migrate` command, I assumed it would work something like `brew migrate <oldname> <newname>`, and would do everything for me.